### DISCUSSION TITLE
help me rq

### DISCUSSION THREAD

**Discussion by Solar-Geaux** [dmMaze/BallonsTranslator#612] [Q&A]
![Screenshot (117)](https://github.com/user-attachments/assets/53a8f3eb-d135-499f-b558-2a560ed2cc20)
I've had this thingy where the text box generates too small for the text, cutting off the top and sometimes the sides, i first thought it was the font (used to  use CC Wild Words) anyways im wondering if anyone else had this thing and has a fix or change i need to do to fix this, i noticed this happening after switching the ocr from mit48px to mangaOCR, i also changed the text detection briefly however i put that back to exactly how it was.

**Comment by dmMaze** [comment]
Please update the program, it should be fixed now. 
And please post these buggy problems in the issue section in the future.
